### PR TITLE
Fix for outdated feed URL stored in user preferences

### PR DESCRIPTION
**Pull request by barijaona**
Also updated copyright strings

### CODE REVIEW

File: src/AppController.m [hunk 1]
```diff
@@ -586,6 +586,25 @@ - (void)updaterWillRelaunchApplication:(SUUpdater *)updater
 	
 }
 
+/* feedURLStringForUpdater:
+ * This is a delegate for Sparkle.framework
```
Comment (josh64x2) [inline src/AppController.m]: This should go in a separate class - something like `ViennaSparkleDelegate.m`. AppController is our most bloated class and we need to trim it down.

**Comment by josh64x2** [comment]
I wish github would notify me about new commits to PRs that I have commented on! Sorry about the delay @barijaona.

**Comment by barijaona** [comment]
No problem, Josh, you are very quick to review

**Comment by barijaona** [comment]
No problem, Josh, you are very quick to review

**Pull request merged**